packages/backend/src/modules/dashboard/dashboard.repository.ts

```diff
@@ -1,27 +1,50 @@
-import { CreateTaskCommandDTO } from 'shared'
+import { ObjectId } from 'mongoose'
+import { CreateTaskCommandDTO, UpdateTaskCommandDTO } from 'shared'
 import DashboardModel from './Dashboard'
 import { Task } from '../task/task'
 
 export class DashboardRepository {
   private readonly _dashboardModel = DashboardModel
 
+  // findIndexInDatabase is looking for index by id in an Array of Objects
+  private findIndexOfDocument(id: string, array: { _id: ObjectId }[]): number {
```

Comment:
index or id? Be consistent

Reply:
I looking for index but to do that I need to use id.